lib/identity_cache/cached/association.rb

```diff
@@ -0,0 +1,43 @@
+module IdentityCache
+  module Cached
+    class Association # :nodoc:
```

Comment:
I don't think that's possible due to where `Builder` being under `Cached`.

Reply:
I didn't mean we should use `private_constant` on each of these constants, since that would make them private to the `Cached` module, rather than to the `IdentityCache` library in general.  I meant we should make the `Cached` module private, and by doing so make everything under it private to outside of the library.

The only place `Cached` is referenced with an absolute reference is test/test_helper.rb in a the `build_cached` method that we could just remove, since it just adds unnecessary indirection anyways.

Reply:
Ah, I was misunderstanding how `private_constant` works. I've made `Cached` private. 👍